Docs/Sample.plist

```diff
@@ -1057,6 +1059,8 @@
 			<integer>17760515</integer>
 			<key>SecureBootModel</key>
 			<string>Default</string>
+			<key>ToggleSipMask</key>
```

Comment:
To be honest, I do not believe there is any sense for this value to be in the config. It will get dropped in the future when this migrates to a separate driver anyway.

Reply:
Not convinced by this - in that if someone was using Catalina and using ToggleSip in OC, they would definitely prefer to set 0x77 here instead of 0x7F.

Reply:
It is not about conviction, this thing will not be supporred once SIP control moves to a separate driver, and thus there is no adequate need to add such an option for soon removal.

Reply:
Okay - will remove - as I've re-done the tool (as should have been anyway) - Catalina users and the like could always set up a Tools entry.